spec.bs

```diff
@@ -280,9 +280,16 @@ To <dfn>look up penultimate redemption</dfn> given an [=/origin=] |issuer| and a
 1. If [=redemptionTimes=][(|issuer|,|topLevel|)] [=map/exists=], let |penultimateRedemption| be the |penultimateRedemption| field of the tuple [=redemptionTimes=][(|issuer|,|topLevel|)].
 1. Return |penultimateRedemption|.
 
-To <dfn>look up the key commitments</dfn> for a given [=/origin=] |issuer| and an [=/origin=] |topLevel|, run the following steps:
+An user agent has <dfn>pstKeyCommitments</dfn>, a [=map=] where the keys are an [=/origin=] |issuer|, and the values are a [=map=] |commitment|. Populating |pstKeyCommitments| is [=implementation-defined=].
 
-1. TODO
+To <dfn>look up the key commitments</dfn> for a given [=/origin=] |issuer|, run the following steps:
+
+1. If [=pstKeyCommitments=][|issuer|] does not [=map/exist=], return false.
+1. Let |issuerKeys| be [=pstKeyCommitments=][|issuer|].
+1. Let |issuerVersionKeys| be false.
```

Comment:
Is this always a boolean value? It doesn't sound boolean (I'd suggest `hasIssuerVersionKeys` or something like that if it was). Otherwise I'd suggest [null](https://infra.spec.whatwg.org/#nulls).